src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -93,6 +94,12 @@ public static String getCurrentBallerinaVersion() {
                     setInstallerVersion(installerVersionFilePath);
                 }
             }
+            String ballerinaVersionFilePath = OSUtils.getBallerinaVersionFilePath();
```

Comment:
We are not using this variable right?  Shall we avoid ?

Reply:
Updated the PR. OSUtils.getBallerinaVersionFilePath() has been used in a below step as well.